GameData/002_CommunityPartsTitles/Localization/CPT_SquiggsySpaceResearch.cfg

```diff
@@ -39,4 +39,6 @@ Localization
     }
 }
 
-
+// Conditional Parts included with the mod
+@PART[microArgon]:NEEDS[SquiggsySpaceResearch,NearFuturePropulsion]:AFTER[SquiggsySpaceResearch] { @title = Ar-03-k4 Octagonal Argon Container (Micro)  }    // Micro Argon Container
+@PART[microCapacitor]:NEEDS[SquiggsySpaceResearch,NearFutureElectrical]:AFTER[SquiggsySpaceResearch] { @title = ZZ-200 Micro Capacitor (SSR) }               // Micro Capacitor
```

Comment:
:AFTER also works as :NEEDS, so if there is :AFTER[SquiggsySpaceResearch], the patch will not be working without the SquiggsySpaceResearch

Reply:
Awesome! Adjusted accordingly.